hooks/PythNetwork/useScalingFactor.ts

```diff
@@ -0,0 +1,33 @@
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
+import { determineVotingPowerType } from "@hooks/queries/governancePower";
+import { useRealmQuery } from "@hooks/queries/realm";
+import { PythClient } from "@pythnetwork/staking";
+import { useConnection } from "@solana/wallet-adapter-react";
+import { useAsync } from "react-async-hook";
+
+
+export default function useScalingFactor() : number | undefined {
+    const realm = useRealmQuery().data?.result
```

Comment:
If we are using realm for only it's publicKey, can we use `useSelectedRealmPubkey`? I am assuming this returns the same publicKey.

Reply:
Done